packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -4,218 +4,230 @@ title: OAuth 2.0 and OpenID Connect Overview
+* [OIDC](/docs/reference/api/oidc/) extends OAuth 2.0 with user authentication and Single Sign-On (SSO) functionality. It also enables you to retrieve and store authentication information about your end users. It also defines several OAuth 2.0 scopes to enable applications to access user profile information.
 
-For most of your app auth requirements, we recommend that you use the OAuth 2.0 and OIDC protocols through the different solutions Okta provides, as outlined in [Redirect authentication vs. embedded authentication](/docs/concepts/redirect-vs-embedded/). To get started with auth implementation and find sample apps, see [Sign users in](/docs/guides/sign-in-overview/).
+Okta recommends using one of its authentication deployment models for your application's authentication needs. They each abstract over the OAuth 2.0 and OIDC protocols, so you don't have to use them directly. To get started and to find sample apps, see [Sign users in](/docs/guides/sign-in-overview/).
 
-> **Note**: If you require a completely custom app setup and workflow with direct access control to your Okta org and app integrations, then you can use the [Authentication API](/docs/reference/api/authn/). This API underpins both the Okta [Redirect](/docs/guides/sign-into-web-app-redirect/) and [Embedded](/docs/guides/embedded-siw/) Sign-In Widget, and [Auth JS](/docs/guides/auth-js/) SDKs.
+> **Tip**: Use the [Authentication API](/docs/reference/api/authn/) if you require a custom app setup and workflow with direct access to your Okta org and app integrations. This API underpins both the Okta [redirect model](/docs/guides/sign-into-web-app-redirect/), [Embedded Sign-In Widget](/docs/guides/embedded-siw/), and [Auth JS](/docs/guides/auth-js/) SDKs.
 
 ### OAuth 2.0
 
 OAuth 2.0 is a standard that apps use to provide client applications with access. If you would like to grant access to your application data in a secure way, then you want to use the OAuth 2.0 protocol.
 
 The OAuth 2.0 spec has four important roles:
 
-* **authorization server**: The server that issues the access token. In this case Okta is the authorization server.
-
-* **resource owner**: Normally your application's end user that grants permission to access the resource server with an access token.
-
-* **client**: The application that requests the access token from Okta and then passes it to the resource server.
-
-* **resource server**: Accepts the access token and must verify that it's valid. In this case, this is your application.
+* **Client**: The application that wants to access some data.
+* **Resource Server**: The API or application that stores the data the client wants to access.
+* **Resource Owner**: The owner of the data in the resource server. For example, you are the owner of your Facebook profile.
```

Comment:
I might suggest the data is 'on' the server rather than 'in' the server, but I guess either works.
```suggestion
* **Resource Owner**: The owner of the data in the resource server. For example, you're the owner of your Facebook profile.
```